src/grt/src/Grid.cpp

```diff
@@ -132,14 +114,12 @@ std::pair<Grid::Tile, Grid::Tile> Grid::getBlockedTiles(
                                    // the center of the tile where it is inside
 
   // Get x and y indices of first blocked tile
-  first_tile.x = (lower.x() - (getTileWidth() / 2)) / getTileWidth();
-  first_tile.y = (lower.y() - (getTileHeight() / 2)) / getTileHeight();
+  first_tile.setX((lower.x() - (getTileWidth() / 2)) / getTileWidth());
```

Comment:
suggest combine setX and setY to setLoc because they will never be set individually